### ISSUE TITLE
Exclude page templates

### ISSUE THREAD

**Issue by grappler** [cedaro/grunt-wp-i18n#67]
I was trying to exclude a page template from the makepot. After looking at the code I realized it is not possible.

It would be nice if the code would first check if the file excluded before running
`$potextmeta->append( $dir . '/' . $file, $output, array( 'Template Name' ) )`

**Comment by bradyvercher** [comment]
The PHP code really is a bit of a mess and it would probably require a decent bit of duplication/refactoring to make that work. Could you use the [processPot](https://github.com/cedaro/grunt-wp-i18n/blob/develop/docs/makepot.md#optionsprocesspot) callback to remove that string?

**Comment by grappler** [comment]
I just needed to generate the pot file once so I just deleted the page template temporarily. I reported the issue so that the next person might find out earlier of the issue.

I would be interested to know how you would remove a string. I saw the code in [remove-package-metadata.md](https://github.com/cedaro/grunt-wp-i18n/blob/develop/docs/examples/remove-package-metadata.md) but it was a bit complex for me to understand staight away. Is the pot variable documented somewhere.

**Comment by bradyvercher** [comment]
Yeah, that example is a little complex without being familiar with the code, but that variable is an object representation of the POT file created by the gettext-parser module. The [structure is documented here](https://github.com/andris9/gettext-parser#data-structure-of-parsed-mopo-files).
